src/scheduler/gc_work.rs

```diff
@@ -253,8 +253,7 @@ impl<VM: VMBinding> GCWork<VM> for EndOfGC {
 
         mmtk.plan.base().set_gc_status(GcStatus::NotInGC);
 
-        // Reset the triggering information.
-        mmtk.plan.base().reset_collection_trigger();
```

Comment:
This seems to be a merge issue. This should not be removed.

Reply:
I accidentally removed this right before I commit and push. It was there when I triggered the assert failure